HW02_A_Graph-Fails.Rmd

```diff
@@ -1,87 +1,90 @@
----

```

Comment:
This is technically this is the document you need to put the code into. Then you "knit" the rmd in Rstudio and it creates an md file that you commit to git